### PR TITLE
Update deps, avoid using `parity-common` fork

### PR DESCRIPTION
**Pull request by sergeyboyko0791**

### CODE REVIEW

File: util/keccak-hasher/Cargo.toml [hunk 1]
```diff
@@ -9,4 +9,5 @@ license = "GPL-3.0"
 ethereum-types = { version = "0.4", default-features = false, features = ["std", "serialize"] }
 tiny-keccak = "1.4.2"
 hashdb = { git = "https://github.com/artemii235/parity-common" }
-plain_hasher = { git = "https://github.com/artemii235/parity-common" }
\ No newline at end of file
+plain_hasher = { git = "https://github.com/artemii235/parity-common" }
```
Comment (artemii235) [inline util/keccak-hasher/Cargo.toml]: As `avoid using parity-common fork` is mentioned, I suppose it's worth trying to remove its usage here too.

**Review by artemii235 — CHANGES_REQUESTED** [review]
Thanks for the PR! I have one change proposal.

File: util/keccak-hasher/Cargo.toml (hunk 1 above)
Comment (sergeyboyko0791) [reply util/keccak-hasher/Cargo.toml]: Missed this. I removed `keccak-hasher` at all. Please check

**Review by artemii235 — APPROVED** [review]
👍

**Pull request merged**